src/app/components/mnoe-dashboards-list/mnoe-dashboards-list.html

```diff
@@ -0,0 +1,91 @@
+<mno-widget icon="fa-bar-chart" is-loading="$ctrl.dashboards.loading">
+  <mno-widget-header>
+    &nbsp;{{ 'mnoe_admin_panel.dashboard.dashboard_list.widget.list.title' | translate}}
+  </mno-widget-header>
+  <mno-widget-body class="no-padding">
+    <div>
+      <table st-pipe="$ctrl.callServer" class="table table-striped" st-table="$ctrl.dashboards.list">
+
+        <!-- Dashboard list header -->
+        <thead>
+          <tr>
+            <th st-sort="name" class="col-lg-3" translate>mnoe_admin_panel.dashboard.dashboard_list.widget.list.name</th>
+            <th class="col-lg-3" translate>mnoe_admin_panel.dashboard.dashboard_list.widget.list.currency</th>
+            <th st-sort="updated_at" class="col-lg-3" translate>mnoe_admin_panel.dashboard.dashboard_list.widget.list.updated_at</th>
+            <th st-sort="created_at" class="col-lg-3" translate>mnoe_admin_panel.dashboard.dashboard_list.widget.list.created_at</th>
+          </tr>
+        </thead>
+
+        <!-- Dashboard list -->
+        <tbody ng-repeat="dashboard in $ctrl.dashboards.list">
+          <tr ng-click="$ctrl.toggleDashboard(dashboard)">
+            <td title="{{ dashboard.full_name }}">
+              <span>
+                <i class="fa" ng-class="{ 'fa-caret-right': !dashboard.opened, 'fa-caret-down': dashboard.opened }"></i>
+              </span>
+              {{ dashboard.name }}
+            </td>
+            <td title="{{ dashboard.full_name }}">{{ $ctrl.currency(dashboard) }}</td>
+            <td title="{{ dashboard.full_name }}">{{ dashboard.updated_at | date: "dd/MM/yyyy \'at\' h:mm a" }}</td>
```

Comment:
Might be good to consider using angular moment here, something like:
```
{{ dashboard.updated_at | amDateFormat: 'L' }}
```
this way the format will be based on locale
More info on the moment pipes here: https://github.com/urish/ngx-moment

Same for the rest of the dates